src/media/js/addon/containers/reviewDetail.js

```diff
@@ -31,15 +33,26 @@ export class AddonReviewDetail extends React.Component {
   }
 
   render() {
-    if (!this.props.addon || !this.props.addon.slug) {
+    const addon = this.props.addon;
+
+    if (!addon || !addon.slug) {
```

Comment:
This makes perfect sense, but it's a really awkward pattern. There's gotta be a better way to do this. Hm.